src/components/resource/_fields/validate-date-range.js

```diff
@@ -109,16 +107,16 @@ const validateWithinPackageRange = (resourceDateRange, packageDateRange) => {
     beginCoverage: packageBeginCoverage,
     endCoverage: packageEndCoverage
   } = packageDateRange;
-  // javascript/moment has no mechanism for "infinite", so we
+  // javascript/dayjs has no mechanism for "infinite", so we
   // use an absurd future date to represent the concept of "present"
-  const present = moment.utc('9999-09-09T05:00:00.000Z');
+  const present = dayjs.utc('9999-09-09T05:00:00.000Z');
   if (packageBeginCoverage) {
-    const beginCoverageDate = moment.utc(resourceDateRange.beginCoverage);
-    const endCoverageDate = resourceDateRange.endCoverage ? moment.utc(resourceDateRange.endCoverage) : present;
+    const beginCoverageDate = dayjs.utc(resourceDateRange.beginCoverage);
+    const endCoverageDate = resourceDateRange.endCoverage ? dayjs.utc(resourceDateRange.endCoverage) : present;
 
-    const packageBeginCoverageDate = moment.utc(packageBeginCoverage);
-    const packageEndCoverageDate = packageEndCoverage ? moment.utc(packageEndCoverage) : moment.utc();
-    const packageRange = moment.range(packageBeginCoverageDate, packageEndCoverageDate);
+    const packageBeginCoverageDate = dayjs.utc(packageBeginCoverage);
+    const packageEndCoverageDate = packageEndCoverage ? dayjs.utc(packageEndCoverage) : dayjs.utc();
+    const packageRange = new DayRange(packageBeginCoverageDate, packageEndCoverageDate);
```

Comment:
Hey @JohnC-80 this is a very very belated review, apologies for that.
It looks like functions for `moment.contains` and `dayjs.isBetween` work differently.
`moment` returns `true` if we check dates that are either the beginning or end date, but `dayjs` will return false.
For example, for when checking if date `2021-01-01` is in range `2021-01-01` to `2021-01-31` `moment` will return true, but `dayjs` will return false.
The `isBetween` plugin has a parameter that defines if begin and end dates should or should not be included: https://day.js.org/docs/en/plugin/is-between

I think since we've been using `moment` and by default it includes start and end date then we should do the same with `dayjs`. Do you have any thoughts?

Reply:
I am working on making this PR up-to-date so we can merge it, just need to confirm with you if we should make the date inclusivity a default behaviour or not

Reply:
Created a PR to handle date ranges correctly https://github.com/folio-org/stripes-components/pull/2443